src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/AccountController.cs

```diff
@@ -127,7 +129,9 @@ public ActionResult Login(string returnUrl)
         /// </summary>
         /// <returns>View to represent</returns>
         [Authorize]
+#pragma warning disable S4261 // Methods should be named according to their synchronicities
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)